pkgs/applications/science/biology/bftools/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, lib, pkgs, fetchurl, unzip, openjdk }:
+
+stdenv.mkDerivation rec {
+  name = "bftools-${version}";
+  version = "5.9.2";
+
+  src = fetchurl {
+    url = "http://downloads.openmicroscopy.org/bio-formats/${version}/artifacts/bftools.zip";
+    sha256 = "02x8p07z7js41kgiblkr9k8v9v4ka8rm7pg3f1vy9s5p6hmpfvnx";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ./* $out/
+    for script in $(find . -maxdepth 1 -perm -111 -type f); do
+      ln -s $out/$script $out/bin/$script
+    done
+  '';
+
+  postFixup = ''
+        for script in "$out"/bin/*; do
+          wrapProgram "$script" --prefix PATH : "${lib.makeBinPath [ openjdk ]}"
+        done
+      '';
+
+  buildInputs = [ unzip pkgs.makeWrapper ];
```

Comment:
replace pkgs with makeWrapper in the arguments in the first line of the file and then you can refer to makeWrapper without `pkgs.`.

Reply:
done thx